modules/image.yml

```diff
@@ -27,3 +27,6 @@ compatibility:
   nuxt: ^2.0.0 || ^3.0.0
   requires:
     bridge: optional
+  versionMap:
+    2.x: latest
```

Comment:
would it make sense for the map to be `[any semver string]: any semver string`?

Reply:
Yes both sides are semver compatible (righthand is basically whatever `npm i pkg@contraint` allows)

Reply:
Good for me!